Sources/WinUIBackend/WinUIBackend.swift

```diff
@@ -1163,6 +1162,259 @@ public final class WinUIBackend: AppBackend {
         }
     }
 
+    public func createPathWidget() -> Widget {
+        WinUI.Path()
+    }
+
+    public func createPath() -> Path {
+        GeometryGroupHolder()
+    }
+
+    public func updatePath(_ path: Path, _ source: SwiftCrossUI.Path, pointsChanged: Bool) {
+        path.strokeStyle = source.strokeStyle
+
+        if pointsChanged {
+            path.group.children.clear()
+            applyActions(source.actions, to: path.group.children)
+        }
+
+        path.group.fillRule =
+            switch source.fillRule {
+                case .evenOdd:
+                    .evenOdd
+                case .winding:
+                    .nonzero
+            }
+    }
+
+    func requirePathFigure(
+        _ collection: WinUI.GeometryCollection,
+        lastPoint: Point
+    ) -> PathFigure {
+        var pathGeo: PathGeometry
```

Comment:
Name this pathGeometry unless there some reason it can't be called that

Reply:
done